.github/workflows/ci.yml

```
@@ -11,12 +11,16 @@ jobs:
     runs-on: ubuntu-latest
     
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@v5
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)